x11-terms/roxterm/roxterm-3.9.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake xdg-utils
+
+DESCRIPTION="A terminal emulator designed to integrate with the ROX environment"
+HOMEPAGE="https://github.com/realh/roxterm"
+SRC_URI="https://github.com/realh/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-3"
+SLOT="1"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-libs/dbus-glib-0.100
```

Comment:
Version restriction can be lifted.

Reply:
What are the reasons for this? Isn't it good practice to at least state the oldest version if there is one?

Reply:
Then you should specify minimum requirements for every dep, not just one. It's just a lot cleaner and easier to maintain this way.